common/src/main/java/xyz/wagyourtail/jsmacros/client/api/helpers/BossBarHelper.java

```diff
@@ -1,23 +1,157 @@
 package xyz.wagyourtail.jsmacros.client.api.helpers;
 
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.gui.hud.ClientBossBar;
 import net.minecraft.entity.boss.BossBar;
+import net.minecraft.text.Text;
 import net.minecraft.util.Formatting;
-
+import xyz.wagyourtail.jsmacros.client.access.IBossBarHud;
 import xyz.wagyourtail.jsmacros.core.helpers.BaseHelper;
 
 import java.util.Locale;
+import java.util.Map;
+import java.util.UUID;
 
 /**
  * @author Wagyourtail
  * @since 1.2.1
  */
-@SuppressWarnings("unused")
 public class BossBarHelper extends BaseHelper<BossBar> {
 
+    private BossBarHelper(UUID uuid, Text name, float percentage, BossBar.Color color, BossBar.Style style){
+        super(new ClientBossBar(uuid, name, percentage, color, style, false, false, false));
+    }
+
+    /**
+     * @since 1.6.5
+     * @param name Title of the BossBar
+     * @param percentage Percentage filled should be between 0 and 1
+     * @param color one of the following: "pink", "blue", "red", "green", "yellow", "purple", "white"
+     * @param style one of the following: "progress", "notched_6", "notched_10", "notched_12", "notched_20"
+     */
+    public BossBarHelper(TextHelper name, float percentage, String color, String style){
+        this(UUID.randomUUID(), name.getRaw(), percentage, BossBar.Color.byName(color.toUpperCase(Locale.ROOT)), BossBar.Style.byName(style.toLowerCase(Locale.ROOT)));
```

Comment:
I'd prefer if you fill the alternate constructor's args in the `FChat#createBossBar`

Reply:
also, maybe not in FChat, I think FWorld would make more sense